components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Extensions/StackTypesEx.cs

```diff
@@ -429,8 +430,6 @@ public static class StackTypesEx {
             switch (mode) {
                 case UaDataChangeTrigger.Status:
                     return DataChangeTriggerType.Status;
-                case UaDataChangeTrigger.StatusValue:
```

Comment:
why did we remove this?

Reply:
StatusValue is the UA default value and null means "default" on our nullable enum version.

Reply:
That's per spec correct. Nevertheless, the user should have the flexibility to explicitly set explicitly StatusValue if they want to. In case the server is not compliant or it's behavior might differ from the null default and explicit StatusValue. This was an explicit requirement from a customer.

Reply:
Ah, ok, so what you are saying is you would like a way to force applying a data change filter, correct?  Yes, that would be a way.  Then we need to change the comparer implementation and treat null different from statusvalue.